bin/sl-run.js

```diff
@@ -31,6 +31,8 @@ config.sendMetrics();
 
 if(!config.clustered) {
   console.log('supervisor running without clustering (unsupervised)');
+} else {
+  config.sendExpressMetrics();
```

Comment:
`sendExpressMetrics` does not have a reference to the config, thus I can't put the check there. I'll do the check in `config.start`.